### DISCUSSION TITLE
Accurate transcription but inaccurate timestamp

### DISCUSSION THREAD

**Discussion by beaudry12087** [deepgram/community#632] [General help]
### Which Deepgram product are you using?

Deepgram API

### Details

We ran the transcription of an mp4 file via the deepgram API. It looks like the transcription was done properly but the timestamps are off. The request id is ```
7311d235-529a-4852-86e0-c0f3219e9a72```

### If you are making a request to the Deepgram API, what is the full Deepgram URL you are making a request to?

/v1/listen?model=nova-2&tiers=nova&language=en-US&callback=https%3A%2F%2Fs.trebble.fm%2FonDeepgramTranscriptionJobStatusChange&filler_words=true&punctuate=true&diarize=true

### If you are making a request to the Deepgram API and have a request ID, please paste it below:

7311d235-529a-4852-86e0-c0f3219e9a72

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.
